dbos/logger.py

```diff
@@ -73,25 +74,28 @@ def config_logger(config: "ConfigFile") -> None:
                 export_timeout_millis=5000,
             )
         )
+        global otlp_handler
         otlp_handler = LoggingHandler(logger_provider=log_provider)
 
         # Attach DBOS-specific attributes to all log entries.
+        global otlp_transformer
         otlp_transformer = DBOSLogTransformer()
 
-        # Direct all logs to OTLP
-        add_otlp_to_all_loggers(otlp_handler, otlp_transformer)
+        # Direct DBOS logs to OTLP
+        dbos_logger.addHandler(otlp_handler)
+        dbos_logger.addFilter(otlp_transformer)
 
 
-def add_otlp_to_all_loggers(
-    otlp_handler: LoggingHandler, otlp_transformer: DBOSLogTransformer
-) -> None:
-    root = logging.root
+def add_otlp_to_all_loggers() -> None:
+    if otlp_handler is not None and otlp_transformer is not None:
```

Comment:
Should we assert both of them to be not None? Otherwise, we might silently fail.